geoportailv3/templates/index.html

```diff
@@ -196,7 +197,9 @@ <h2 translate>infos</h2>
     <script src="${request.route_url('deps.js')}"></script>
     <script src="${request.static_url('geoportailv3:static/js/main.js')}"></script>
     <script src="${request.static_url('%s/ngeo/utils/watchwatchers.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/proj4/dist/proj4-src.js' % node_modules_path)}"></script>
 % else:
+    <script src="${request.static_url('%s/proj4/dist/proj4-src.js' % node_modules_path)}"></script>
```

Comment:
Can I take you up on the offer and merge this as is, and we deal with build.js in separate pull request.